src/utils/validation/validators.js

```diff
@@ -0,0 +1,11 @@
+const required= value => {
+    if (value) return undefined;
+    return "HET HAX";
+}
+
+const maxLenghtCreator = (maxLenght) => (value) => {
+    if (value.lenght > maxLenght) return `Max lenght is ${maxLenght} symbols`;
```

Comment:
Consider rewriting this logic to utilise a ternary operator.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
done